rules/eslint/node/walmart.js

```diff
@@ -7,6 +7,8 @@ module.exports = {
   "rules": {
     // enforce return after a callback
     "callback-return": 2,
+    // disallow require() outside of the top-level module scope
+    "global-require": 0,
```

Comment:
What do you think about this?  In ES5 world, I've seen (and probably used) the following pattern:
```javascript
module.exports = {
  "thing": require("./thing")
  "otherThing": require("./other-thing")
};
```
But since we're moving towards ES6 modulesin general, this use case goes away.  Any other reasons to call it OK?  There is a mild reason to turn the rule on, as it isn't clear from the head of the file what dependencies the module takes.